docker/build_artifacts/deep_learning_container.py

```diff
@@ -75,8 +89,10 @@ def query_bucket():
     region = _retrieve_instance_region()
 
     if instance_id is not None and region is not None:
-        url = ("https://aws-deep-learning-containers-{0}.s3.{0}.amazonaws.com"
-               "/dlc-containers.txt?x-instance-id={1}".format(region, instance_id))
+        url = (
+            "https://aws-deep-learning-containers-{0}.s3.{0}.amazonaws.com"
```

Comment:
Do we need to continue to have this file on this repo? Can we remove it?

Reply:
We are planning a clean up of this repo which involves revisting/updating some tests. As of now this file is used while building a docker image (mimic DLC) and host tfs model server in some tests. We will keep it for now.